<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>prod_line</class>
 <widget class="QWidget" name="prod_line">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>64</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>20</width>
    <height>25</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,1,2">
   <item>
    <widget class="QLabel" name="prod_name">
     <property name="text">
      <string/>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="prod_quant">
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="result">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
